### PR TITLE
Prevent XSS attacks by enabling a cleanup method

### PR DESCRIPTION
**Pull request by brunobord**
## Review

* [x] Tests
* [x] Docs/comments
* [x] `CHANGELOG.rst` Updated
* [ ] Delete your branch

### CODE REVIEW

File: formidable/serializers/defaults.py [hunk 1]
```diff
@@ -17,7 +18,16 @@ class Meta:
         fields = ('value',)
 
     def to_internal_value(self, data):
-        return {'value': data}
+        # NOTE: for some reason, data equals to the value.
+        # FIXME: investigate a bit further why.
```
Comment (syldlb) [inline formidable/serializers/defaults.py]: Do you want to do it in this PR or it's a reminder for later?

File: docs/source/security.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
+==============
+Security setup
+==============
+
+As any other web application, Django Formidable might be targeted by pirates and would try to inject SQL or malicious code through Javascript or any other XSS method.
```
Comment (syldlb) [inline docs/source/security.rst]: `who` would try?

Comment (brunobord) [reply docs/source/security.rst]: ah yes. missing word.

File: formidable/serializers/defaults.py (hunk 1 above)
Comment (brunobord) [reply formidable/serializers/defaults.py]: I'm really not sure... it goes beyond the scope of this PR, and I wish I knew better about DRF to figure out why we have to convert to dict. As you can see, on master, it's already converted in the `to_internal_value` method.

File: demo/demo/security.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    """
+    return bleach.clean(
+        obj,
+        BLEACH_VALID_TAGS,
+        BLEACH_VALID_ATTRS,
+        BLEACH_VALID_STYLES,
+    )
+
+
+def clean_alert(input_string):
```
Comment (r2mi-pdoc) [inline demo/demo/security.py]: @brunobord what is the purpose of this function ?
I think it should be better to implement a regex-based bleaching method (to identified common XSS payload) to be more effective and more generic.

`bleach` library filters authorized HTML tags, attributes and css-style, then another function like `clean_content()` could detect XSS payload on content that passed `bleach.clean()`.

Comment (brunobord) [reply demo/demo/security.py]: this function belongs to the **demo** site, which is only used in tests. It's here as a "proof of concept" that any 3rd party integrator may use any kind of function they want to clean their own content.

It's also used in this test: https://github.com/peopledoc/django-formidable/pull/378/files#diff-c8e5a0cefee15566bc22894bcf61a925R40-R44 to show how changing the `DJANGO_FORMIDABLE_SANITIZE_FUNCTION` settings and pointing it at an existing function is changing the content passed through it.

**Pull request merged**